plugins/modules/remote_management/redfish/wdc_redfish_command.py

```diff
@@ -0,0 +1,265 @@
+    type: str
+  ioms:
+    description:
+      - List of IOM FQDNs for the enclosure.  Must include this or baseuri
+    type: list
+    elements: str
+  username:
+    description:
+      - User for authentication with OOB controller
+    type: str
+  password:
+    description:
+      - Password for authentication with OOB controller
+    type: str
+  auth_token:
+    description:
+      - Security token for authentication with OOB controller
+    type: str
+  timeout:
+    description:
+      - Timeout in seconds for URL requests to OOB controller
+    default: 10
+    type: int
+  update_image_uri:
+    required: false
+    description:
+      - The URI of the image for the update
+    type: str
+  update_creds:
+    required: false
+    description:
+      - The credentials for retrieving the update image
+    type: dict
+    suboptions:
+      username:
+        required: false
+        description:
+          - The username for retrieving the update image
+        type: str
+      password:
+        required: false
+        description:
+          - The password for retrieving the update image
+        type: str
+requirements:
+  - dnspython (2.1.0 for Python 3, 1.16.0 for Python 2)
+notes:
+  - In the inventory, you can specify baseuri or ioms.  See the EXAMPLES section.
+  - ioms is a list of FQDNs for the enclosure's IOMs.
+
+
+author: Mike Moerk (@mikemoerk)
+'''
+
+EXAMPLES = '''
+- name: Firmware Activate (required after SimpleUpdate to apply the new firmware)
+  community.general.wdc_redfish_command:
+    category: Update
+    command: FWActivate
+    ioms: "{{ ioms }}"
+    username: "{{ username }}"
+    password: "{{ password }}"
+
+- name: Firmware Activate with individual IOMs specified
+  community.general.wdc_redfish_command:
+    category: Update
+    command: FWActivate
+    ioms:
+      - iom1.wdc.com
+      - iom2.wdc.com
+    username: "{{ username }}"
+    password: "{{ password }}"
+
+- name: Firmware Activate with baseuri specified
+  community.general.wdc_redfish_command:
+    category: Update
+    command: FWActivate
+    baseuri: "iom1.wdc.com"
+    username: "{{ username }}"
+    password: "{{ password }}"
+
+
+- name: Update and Activate (orchestrates firmware update and activation with a single command)
+  community.general.wdc_redfish_command:
+    category: Update
+    command: UpdateAndActivate
+    ioms: "{{ ioms }}"
+    username: "{{ username }}"
+    password: "{{ password }}"
+    update_image_uri: "{{ update_image_uri }}"
+    update_creds:
+      username: operator
+      password: supersecretpwd
+'''
+
+RETURN = '''
+msg:
+    description: Message with action result or error description
+    returned: always
+    type: str
+    sample: "Action was successful"
+'''
+
+from ansible_collections.community.general.plugins.module_utils.wdc_redfish_utils import WdcRedfishUtils
+try:
+    from dns import resolver
+    DNS_AVAILABLE = True
+except ImportError:
+    DNS_AVAILABLE = False
```

Comment:
Is this still needed?

Reply:
Indeed it is not.  Removed in [4d7eb7b8f03a0087b35d961b8090894ff8bcba6a](https://github.com/ansible-collections/community.general/pull/4885/commits/4d7eb7b8f03a0087b35d961b8090894ff8bcba6a)